ui/src/settings/components/playground_flyout/PlaygroundFlyout.js

```diff
@@ -76,8 +74,8 @@ const PlaygroundFlyout = ({ onClose }) => {
 
   return (
     <EuiFlyout
-      className="euiFlyout--playgroundFlyout"
```

Comment:
The custom css styles no longer work with the newer version of Elastic UI. Besides, using the size "s" of the Flyout works just as fine (this is currently used in Turing).